content/en/blog/baremetal_performance/index.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "How to achieve baremetal performance inside your virtual machine"
+date: 2022-05-19T08:00:00Z
+draft: false
+contributors: [Cloud Hypervisor Team]
+---
+
+There are use cases for which a workload needs to access specific hardware such
+as accelerators, GPU or network adapters to reach high performance. And since
+these workloads run inside virtual machines (VM) for security reasons, the
+challenge is to make this hardware available from within the VM, but without
+degrading the performance that can be achieved from baremetal.
+
+### VFIO, the first choice

```

Comment:
Since you're using title case for the headings it might be best if you're consistent.

Reply:
By first do you mean ideal or 1st vs 2nd? If the former maybe it's best to use that word instead.

Reply:
Ideal is better.